sdk/build-scripts/package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "@mysten/build-scripts",
+	"private": true,
+	"author": "Mysten Labs <build@mystenlabs.com>",
+	"description": "Build scripts for @mysten packages",
+	"version": "0.0.0",
+	"license": "Apache-2.0",
+	"engines": {
+		"node": ">=16"
+	},
+	"scripts": {
+		"prettier:check": "prettier -c --ignore-unknown .",
+		"prettier:fix": "prettier -w --ignore-unknown .",
+		"eslint:check": "eslint --max-warnings=0 .",
+		"eslint:fix": "pnpm run eslint:check --fix",
+		"lint": "pnpm run eslint:check && pnpm run prettier:check",
+		"lint:fix": "pnpm run eslint:fix && pnpm run prettier:fix",
+		"build": "tsc --build"
+	},
+	"devDependencies": {
+		"esbuild": "^0.18.5",
+		"ts-node": "^10.9.1",
```

Comment:
Is this used?

Reply:
oh, moved this to the sdk (to run the script) I can remove it from here